solver/src/main.rs

```diff
@@ -168,6 +169,21 @@ struct Arguments {
     /// The slippage tolerance we apply to the price quoted by Paraswap
     #[structopt(long, env, default_value = "10")]
     paraswap_slippage_bps: usize,
+
+    /// The authorization for the archer api. If set the archer network will be used to submit
+    /// settlement transactions. If not set then transactions are sent out the normal way through
+    /// the node.
+    #[structopt(long, env)]
+    archer_authorization: Option<String>,
+
+    /// The maximum time we spend trying to settle a transaction through the archer network before
+    /// going to back to solving.
+    #[structopt(
+        long,
+        default_value = "120",
```

Comment:
Maybe 30 secs is a better default, as the price can change a lot in 2 mins.

Reply:
Agree that 2 minutes is too long. Maybe we can aim for the same as "targetConfirmationTime"?

Reply:
If the price changes so that the settlement becomes impossible then we cancel early anyway. This is just the maximum time we wait while the settlement would continue to simulate. For the public mempool this time is implicitly 5 minutes because that is what ethcontract sets by default.
With 30 seconds I am bit worried that there might be some delays with how the transaction gets propagated to the mining pools that make this not enough time. But I'm not sure what the best value is. Maybe something we need to test. Also wouldn't matter if we had this more advanced settlement - solution workflow that we talked about.